wurst/__legacy__/trolls/thief/Jump.wurst

```diff
@@ -6,7 +6,9 @@ import Orders
 import UnitExtensions
 
 constant let JUMP_ID = 'A04G'
-constant let JUMP_RANGE = 600.
+constant let RANGE_MAX = 600.
```

Comment:
Use same name format as `TARGET_SEARCH_MINRANGE`, which should be `TARGET_SEARCH_MIN_RANGE`.